bin/install.js

```diff
@@ -4,20 +4,12 @@ console.log("Installing SCope Bind Server")
 shell.cd('opt/scopeserver/bindserver')
 shell.exec('npm install');
 shell.cd('../../../')
-var execSync = require('child_process').execSync;
```

Comment:
Is this a dependency we can remove now? Great!!

Reply:
No, unfortunately we still need to use it to run commands with `exec`